From e3c4228717593de4ac1e67d4b9aed1239943dbae Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Tue, 2 Feb 2021 17:55:31 +0800
Subject: [PATCH] arm64: dts: rockchip: Add SYS_STATUS_DUALVIEW for
 rk3566/rk3568

The DDR freq should be fixed in dual display mode.

Change-Id: I2383d415f93a52366f762e9480b54451a75f1ad5
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3566.dtsi | 1 +
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566.dtsi b/arch/arm64/boot/dts/rockchip/rk3566.dtsi
index 846c08ba2632..42bf65ed538b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566.dtsi
@@ -26,6 +26,7 @@
 		SYS_STATUS_BOOST        1056000
 		SYS_STATUS_ISP          1056000
 		SYS_STATUS_PERFORMANCE  1056000
+		SYS_STATUS_DUALVIEW     1056000
 	>;
 };
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index be87f241bb64..16f8b81dfd6d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -1983,6 +1983,7 @@
 			SYS_STATUS_BOOST        1560000
 			SYS_STATUS_ISP          1560000
 			SYS_STATUS_PERFORMANCE  1560000
+			SYS_STATUS_DUALVIEW     1560000
 		>;
 		auto-min-freq = <324000>;
 		auto-freq-en = <1>;
-- 
2.35.3

